### ISSUE TITLE
ATOM very Hight CPU usage (more than 100%)

### ISSUE THREAD

**Issue by damiendevienne** [atom/atom#10573]
Hello, 
I read all comments related to this issue, but could not solve it in my case. 
I simply installed atom on my Ubuntu 15.04 from the deb package downloaded from the web site. 

Even if I do nothing on the code I am editing, the CPU usage is around 110% for  atom. 

I am simply typing `atom` on the command line followed by the name of the file I want to edit. 

Running in safe mode has no effect. 

Thanks for you help

**Comment by mosbth** [comment]
Debian, Atom 1.4.0, installed .

This happens to me when I create an empty directory and start Atom from the terminal, like this.
```
$ mkdir empty
$ cd empty
$ atom .
```

Atom consumes 100% of the CPU.

When I create a Git-repository in the empty dir, and restarts Atom, it works as expected.
```
# Still being in the empty directory
$ git init
$ atom .
```

**Comment by 50Wliu** [comment]
Do either of you have a `.git` folder in your home folder?

**Comment by mosbth** [comment]
Yes, I have. I do not know why though...

**Comment by 50Wliu** [comment]
> I do not know why though...

:grimacing: 

In that case, this issue is most likely a duplicate of #6437 and has been fixed in the 1.5.0 betas.

**Comment by mosbth** [comment]
Yep, I removed the `$HOME/.git` folder and it now works as expected. Thanks!

**Comment by damiendevienne** [comment]
Hi, 
I removed my .git folder in the home folder. It was better for 30 minutes or so, but now atom is back to 100% CPU, making it impossible to use.  
By the way, I don't understand why a git folder on a computer has something to do with a program installed with a deb package... 
Any help?
Cheers,

**Comment by mosbth** [comment]
@damiendevienne the reason why I tried with the .git-golder was that I have used this version of Atom successfully for a week (without problem) and it was this empty directory that gave me problem. One thing that differed between the empty directory and the others where I successfully use Atom, was that they are all Git-repo. So I just tried to create a .git-folder and it instantly made the trick for me.

Atom makes use of the .git-folder, and other Git-file like .gitignore, so I guessed it had something to do with the issue. I did a lucky guess since it solved my troubles.

**Comment by damiendevienne** [comment]
I really understand nothing to this. 
I just want to use atom because it seems to be a good tool, and I end up having to delete folders on my computer. This sounds weird, no?  
...I went back to geany... snif...

**Comment by 50Wliu** [comment]
@damiendevienne The problem is that Atom tries to find .git folders to mark your files as added/deleted/modified, but a .git folder in your home directory messes that up as it then has to search every single file recursively :scream:.

You can try installing 1.5.0-beta1 which should fix your CPU issue.

**Comment by AlexanderElias** [comment]
Hello @50Wliu ,

I am also having high CPU usage 100% or more. I disable all packages including atoms except settings. Also tried it with safe mode. Tried using the default github folder and using a custom code folder. Made sure that there was no .git .gitignore folders/files.

Ubuntu 16.04 with KDE Plasma 5.10
Linux Kernel 4.10.0-35-generic
Node v6.11.3
Atom 1.20.1

**Comment by 50Wliu** [comment]
HI @AlexanderElias!  Apologies for the troubles.  You can temporarily disable the `github` package until Atom 1.21 is released, or upgrade to Atom 1.21-beta.  That should bring down your CPU usage; if it doesn't, please open a new issue and I'll try to look into it.

**Comment by AlexanderElias** [comment]
Thanks for the quick response.

I actually disabled all the plugins including the github. I will give 1.2.1 a try and report back.

**Comment by AlexanderElias** [comment]
Just tested 1.2.1 and even with all the plugins (except for settings) disabled I am still getting around 100% cpu usage.

**Comment by Ben3eeE** [comment]
> Ubuntu 16.04 with KDE Plasma 5.10

This is a bug with the current version of Electron we are on /cc: https://github.com/electron/electron/issues/8455

**Comment by AlexanderElias** [comment]
I disabled the global menu in KDE and reset PC and CPU was still 100%. Although I am experiencing a new issue that might be related. The local or regular atom menu wont show up now lol.

Assuming that electron is the issue the claimed fix in electron is version 1.7.6 could we expect to see it updated in atom any time soon?

**Comment by Ben3eeE** [comment]
@AlexanderElias It seems that this fix was backported to Electron 1.6.12. We upgraded to 1.6.14 on master today and intend to release a hotfix with the upgrade. I am checking with @ungb to see if we can verify that this has been fixed, hopefully it will be and the fix will reach you in the next hotfix release 🙂

**Comment by ungb** [comment]
I'm trying to test the 100% cpu on KDE but I don't see this happening for me. @AlexanderElias does Atom run at 100% when running in safe mode? `atom --safe`.  

I am trying this on atom 1.21.0.  If I can get a repro of the issue I'll go ahead and verify if it's fixed off master.

**Comment by AlexanderElias** [comment]
Clean install and removed my .atom dir. Tried safe mode. Reset KDE window settings. Still getting 100% also still not able to get the atom menu to work. I tried toggling it on and off in settings and using the alt key to show do you think this could be connected?

![screenshot_20171011_171857](https://user-images.githubusercontent.com/6393261/31473589-64b70b64-aeaa-11e7-9b63-e386bbaca255.png)

**Comment by Ben3eeE** [comment]
@AlexanderElias We managed to get a repro for this and can confirm that it was fixed in a newer version of Electron. Upgrading Electron is scheduled for the next hotfix release. So this should be fixed on Atom 1.21.1.

Please let us know if you have any problems after upgrading to Atom 1.21.1 :bowing_man:

**Comment by af6140** [comment]
I am getting similar issue here too, just upgrade to atom 1.21.1

strace -f -p pidofatom show it's working on some git object 

{code}
[pid  9199] <... stat resumed> {st_mode=S_IFDIR|0775, st_size=52, ...}) = 0
[pid  9199] open("/home/dwang/pipeline/aws/tf-ent_rds_security_groups_module/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.git/objects/94", O_RDONLY|O_NONBLOCK|O_DIRECTORY|O_CLOEXEC <unfinished ...>
[pid  9201] <... stat resumed> {st_mode=S_IFREG|0775, st_size=424, ...}) = 0
[pid  9201] stat("/home/dwang/pipeline/aws/tf-ent_rds_instance_module/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/1d1bc6ce99275cb702c62eb81e2a7a41/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.git/hooks/pre-commit.sample",  <unfinished ...>
[pid  9199] <... open resumed> )        = 93
[pid  9199] fstat(93, {st_mode=S_IFDIR|0775, st_size=52, ...}) = 0
[pid  9199] getdents(93,  <unfinished ...>
[pid  9201] <... stat resumed> {st_mode=S_IFREG|0775, st_size=1642, ...}) = 0
[pid  9199] <... getdents resumed> /* 3 entries */, 32768) = 112
[pid  9201] stat("/home/dwang/pipeline/aws/tf-ent_rds_instance_module/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/1d1bc6ce99275cb702c62eb81e2a7a41/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.git/hooks/prepare-commit-msg.sample",  <unfinished ...>
[pid  9199] getdents(93, /* 0 entries */, 32768) = 0
[pid  9199] madvise(0x221801aa7000, 282624, MADV_DONTNEED) = 0
[pid  9199] close(93)                   = 0
[pid  9201] <... stat resumed> {st_mode=S_IFREG|0775, st_size=1239, ...}) = 0
[pid  9199] stat("/home/dwang/pipeline/aws/tf-ent_rds_security_groups_module/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.git/objects/94/1f88c56ef5c8279a4e411d697350be4853f276",  <unfinished ...>
[pid  9201] stat("/home/dwang/pipeline/aws/tf-ent_rds_instance_module/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/1d1bc6ce99275cb702c62eb81e2a7a41/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.git/hooks/pre-push.sample",  <unfinished ...>
[pid  9199] <... stat resumed> {st_mode=S_IFREG|0444, st_size=93, ...}) = 0
[pid  9199] stat("/home/dwang/pipeline/aws/tf-ent_rds_security_groups_module/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.git/objects/9f",  <unfinished ...>
[pid  9201] <... stat resumed> {st_mode=S_IFREG|0775, st_size=1348, ...}) = 0
[pid  9201] stat("/home/dwang/pipeline/aws/tf-ent_rds_instance_module/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/1d1bc6ce99275cb702c62eb81e2a7a41/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/test/fixtures/.terraform/modules/f8ee731087f81c9cb4aed2fc9cb44131/.git/hooks/pre-rebase.sample",  <unfinished ...>
[pid  9199] <... stat resumed> {st_mode=S_IFDIR|0775, st_size=52, ...}) = 0
[pid  9199] open("/home/dwang/pipeline/aws/tf-ent_rds_security_groups_module/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/mo^C5355a16d7da5cabb1d54601b0c30b129/test/fixtures/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.terraform/modules/5355a16d7da5cabb1d54601b0c30b129/.git/objects/9f/b7700767078f1d5bf98006fe390a390f140bf3", strace: Process 9174 detached
{code}
